CppCustomVisualizer2/dll/ChildVisualizer.cpp

```diff
@@ -172,13 +172,20 @@ HRESULT CChildVisualizer::GetItems(
     for (UINT32 i = 0; i < Count; i++)
     {
         CString evalText;
+
+        UINT32 index = StartIndex + i;
+        if (index >= 2)
+        {
+            continue;
+        }
+
```

Comment:
This is impossible due to the condition on line 138. I might instead suggest adding something like this to the precompiled header:

```
#if defined(_PREFAST_)

    #define VSAnalysisAssume(cond, text) __analysis_assume(cond)

#elif defined(_DEBUG)

    #define VSAnalysisAssume(cond, text) ASSERT(cond)

#else

    #define VSAnalysisAssume(cond, text)

#endif
```

And then here add:
```
UINT32 index = StartIndex + 1;
VSAnalysisAssume(index < _countof(itemExprsPtr) && index < _countof(itemExprs) , "Should be impossible: already validated at start of function");
```